scripts/drive-prs.md

```
@@ -0,0 +1,353 @@
+BRANCH=$(gh pr view <NUMBER> --json headRefName --jq '.headRefName')
+gh run list --branch "$BRANCH" --limit 10 \
+  --json name,status \
+  --jq '.[] | select(.name == "Address PR Review Feedback" and (.status == "in_progress" or .status == "queued"))'
+```
+
+If a run is already active → skip, it is already being handled.
+
+If not, summarise the outstanding feedback and delegate:
+
+For a **`copilot-swe-agent[bot]`** PR:
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)